csrc/scheduler/pointwise.cpp

```diff
@@ -822,7 +847,9 @@ void schedulePointwise(Fusion* fusion, const PointwiseParams* pparams) {
       // Threads
       reference_tv->split(0, kThreadX);
       // Unroll
-      reference_tv->split(0, pparams->unroll_factor);
+      if (pparams->unroll_factor_inner > 1) {
+        reference_tv->split(0, pparams->unroll_factor_inner);
```

Comment:
qq: we are not using `unroll_factor_outer` in this branch, is that expected?

Reply:
Yes, this `else` branch is for 1D scheduler, all IDs are merged into 1 domain, there is no outer dim.